Classes/BITCrashReportTextFormatter.m

```diff
@@ -830,11 +833,16 @@ + (NSString *)bit_formatStackFrame: (BITPLCrashReportStackFrameInfo *) frameInfo
   NSString *imageName = @"\?\?\?";
   NSString *symbolString = nil;
   
-  BITPLCrashReportBinaryImageInfo *imageInfo = [report imageForAddress: frameInfo.instructionPointer];
+  // When on an ARM64 architecture, normalize the address to remove possible pointer signatures
+  uint64_t instructionPointer = lp64 ? (frameInfo.instructionPointer & 0x0000000fffffffff) : frameInfo.instructionPointer;
+  uint64_t untouchedInstructionPointer = frameInfo.instructionPointer;
+  
+  BITPLCrashReportBinaryImageInfo *imageInfo = [report imageForAddress: instructionPointer];
```

Comment:
We might be able to ultimately improve this and move the above logic into PLCrashReporter.